src/renderer/utils.ts

```diff
@@ -2,6 +2,9 @@ export function formatCompressedPercentage (num: number) {
 	return Math.round(num * 100)
 }
 
+/**
+ * @todo rename: bytesToMb
+ */
```

Comment:
What do you thinking about leading with a verb? Like `convertBytesToMb`?

Reply:
Exactly what I had in mind! Thanks for commenting on this, I totes forgot to come back around 😬